stetho/src/main/java/com/facebook/stetho/inspector/protocol/ScreencastDispatcher.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.protocol;
```

Comment:
Let's move this to `com.facebook.stetho.screencast.android`.  We're beginning to separate Stetho internals from Android-specific pieces so that Stetho can one day be used without Android (maybe *grin*)